modules/ROOT/pages/css-styling.adoc

```diff
@@ -51,6 +51,106 @@ search-bar-container {
 |Left and right arrows navigation
 |===
 
+=== API Catalog
+
```

Comment:
I typically suggest having a small text transition between titles and first steps, tables, examples, and so forth. Is there anything useful to say here that might better prepare the audience for consuming the following format of info?

Reply:
Currently this page has explanatory text at the top and then a sequence of titled tables, but I like the idea of adding some useful introduction text for each table, and will try to do that soon.